server/src/main/java/org/opensearch/telemetry/TelemetrySettings.java

```diff
@@ -83,4 +100,12 @@ public void setSamplingProbability(double samplingProbability) {
     public double getSamplingProbability() {
         return samplingProbability;
     }
+
+    public boolean isTracerFeatureEnabled() {
+        return TRACER_FEATURE_ENABLED_SETTING.get(settingz);
```

Comment:
We actually could make those just a fields (like racingEnabled, ...)  - they are node static